rust/agama-dbus-server/src/main.rs

```diff
@@ -4,15 +4,32 @@ pub mod questions;
 
 use agama_network::NetworkService;
 use std::future::pending;
+use log::LevelFilter;
 
 const ADDRESS: &str = "unix:path=/run/agama/bus";
 
 #[async_std::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // be smart with logging and log directly to journal if connected to it
+    if systemd_journal_logger::connected_to_journal(){
```

Comment:
```suggestion
    if systemd_journal_logger::connected_to_journal() {
```

That reminds me, I don't know how to check for formatting problems. How? Should we make it mandatory?

Reply:
well, I again forget to run `rustfmt` before submit. I will try to configure my IDE to do it

Reply:
btw I can add it to CI with something like `find rust/agama-*/src/ | grep '.rs$' | xargs rustfmt --check --edition 2021`